packages/amplication-server/src/core/build/build.controller.ts

```diff
@@ -16,17 +18,25 @@ import { StepNotCompleteError } from './errors/StepNotCompleteError';
 import { StepNotFoundError } from './errors/StepNotFoundError';
 import { CanUserAccessArgs } from './dto/CanUserAccessArgs';
 import { plainToInstance } from 'class-transformer';
-import { MessagePattern, Payload } from '@nestjs/microservices';
-import { CHECK_USER_ACCESS_TOPIC } from 'src/constants';
+import { EventPattern, MessagePattern, Payload } from '@nestjs/microservices';
+import { BUILD_STATUS_TOPIC, CHECK_USER_ACCESS_TOPIC } from 'src/constants';
```

Comment:
Let's get rid of `src/` imports and make it relative

Reply:
change import to relative